pagopa/pn-frontend
# Feat/pn 2404

## Short description
Get privacy page from OneTrust

## List of changes proposed in this pull request
- Added new self hosted js file
- Added some css rules to default.css file
- Changed informativa-privacy.tsx page to load OneTrust page

## How to test
Run landing application, go to informativa-privacy page and check that everything is ok (mobile and desktop)
